client/tailwind.config.js

```diff
@@ -0,0 +1,12 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ['./app/**/*.{js,jsx,ts,tsx}'],
+  theme: {
+    extend: {},
+  },
+  corePlugins: {
+    preflight: false,
```

Comment:
I don't think so. [MUI's guide on Tailwind style library inoperability](https://mui.com/material-ui/guides/interoperability/#tailwind-css) asks us to explicitly remove Tailwind's preflight 'so it can use the MUI's preflight instead'. Using Tailwind's preflight will override MUI's style, e.g., our colours, buttons, font families, etc., that we have set in `Theme`.

Problem is, I'm not even sure if our codebase has MUI's preflight applied. MUI's preflight is the [`<CssBaseline>`](https://mui.com/material-ui/react-css-baseline/) component, but I don't see it anywhere, nor do I know if it is applied automatically. Preflights are usually automatic in CSS frameworks, and based on the quote above, it sounds like MUI's preflight is automatic.

Reply:
Thanks for clarifying!